handlers/goodshit.js

```diff
@@ -1,16 +1,16 @@
-var _      = require('lodash');
-var log4js = require('log4js');
-var format = require('format');
+let _      = require('lodash');
+let log4js = require('log4js');
+let format = require('format');
 
-var LOG = log4js.getLogger('goodshit');
+let LOG = log4js.getLogger('goodshit');
```

Comment:
oh, missed this earlier

for imports and constants use `const` instead of `let`